### PR TITLE
deheader: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

**Comment by maxim-belkin** [comment]
```
usr/bin/tar
-xvqf
deheader-1.6.tar.gz
deheader-1.6/deheader.1

Failed to execute: /usr/bin/tar
```

**Comment by sjackman** [comment]
```ruby
tar = OS.mac? ? "/usr/bin/tar" : "tar"
```

**Comment by sjackman** [comment]
GNU `tar` doesn't seem to have the `-q (--fast-read)` option that macOS has.

**Comment by maxim-belkin** [comment]
```
==> tar -xvqf deheader-1.6.tar.gz deheader-1.6/deheader.1
.
Last 150 lines from /home/ubuntu/.cache/Homebrew/Logs/deheader/01.tar:
2017-04-20 10:30:13 +0000

tar
-xvqf
deheader-1.6.tar.gz
deheader-1.6/deheader.1

tar: invalid option -- 'q'
Try 'tar --help' or 'tar --usage' for more information.
```

File: Formula/deheader.rb [hunk 1]
```diff
@@ -14,16 +14,18 @@ class Deheader < Formula
...
 
   def install
     ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
 
     # Remove for > 1.6
     # Fix "deheader-1.6/deheader.1: Can't create 'deheader-1.6/deheader.1'"
     # See https://gitlab.com/esr/deheader/commit/ea5d8d4
-    system "/usr/bin/tar", "-xvqf", "deheader-1.6.tar.gz",
+    tar = OS.mac? ? "/usr/bin/tar" : Formula["libarchive"].bin/"tar"
+    system tar, "-xvqf", "deheader-1.6.tar.gz",
```
Comment (sjackman) [inline Formula/deheader.rb]: `"-xvqf"` should be `"-xvf"`?

Comment (maxim-belkin) [inline Formula/deheader.rb]: libarchive's tar understands `-q`

**Comment by maxim-belkin** [comment]
Superseded by #2436 (circle was not pulling in the latest updates to the branch)

**Pull request closed without merging**